### PR TITLE
Cleanup and update tests

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

- Fixed a snapshot and some light code cleanup.
- Forgot to commit (last PR) some accessibility cleanup to labels and make sure they exists on the on-boarding (Splash, Tutorial, Pin) screens.

# Related Issues

n/a

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [x] Run prettier: `npm run style-format`
- [x] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: App/components/shared/Pagination.tsx [hunk 1]
```diff
@@ -46,13 +46,17 @@ export const Pagination: React.FC<IPaginationProps> = ({
     }
   }
 
+  // The `TouchableOpacity` prop `disabled` is commented out below because it
```
Comment (amanji) [inline App/components/shared/Pagination.tsx]: Will there ever be the need to disable the back button in lieu of the comment? If not then we can just remove the property along with the note.

Comment (jleach) [reply App/components/shared/Pagination.tsx]: There seems to be a bug where, if I disable a control then enable it the accessibility label does not show. My work around is to just change the colour to transparent - which I don't love. It causes the button to flicker on my phone. It's better to disable it. I was going to update RN in a later release and monitor to see if the problem goes away. The comment is more of a reminder for me to monitor the bug.

Comment (jleach) [reply App/components/shared/Pagination.tsx]: PS: I can remove it in favour of a GitHub issue.

Comment (amanji) [reply App/components/shared/Pagination.tsx]: Let's track it in a GH issue. For now just remove the `disabled` prop altogether and update the `FIXME` comment so devs know that it shouldn't be added.

Comment (jleach) [reply App/components/shared/Pagination.tsx]: Done. Issue #204 created to track.

**Review by amanji — APPROVED** [review]

**Pull request merged**